src/Domain/Tags.hpp

```diff
@@ -161,15 +161,33 @@ struct MappedCoordinates
   using argument_tags = tmpl::list<MapTag, SourceCoordsTag>;
 };
 
+/// \ingroup DataBoxTagsGroup
+/// \ingroup ComputationalDomainGroup
+/// Computes the inverse Jacobian from the source frame to the target frame.
+/// Specifically, \f$\partial x^{\bar{i}} / \partial x^i\f$, where \f$\bar{i}\f$
+/// denotes the source frame and \f$i\f$ denotes the target frame.
+template <size_t Dim, typename SourceFrame, typename TargetFrame>
+struct InverseJacobian : db::SimpleTag {
+  static std::string name() noexcept {
+    return "InverseJacobian(" + get_output(SourceFrame{}) + "," +
+           get_output(TargetFrame{}) + ")";
+  }
+  using type = ::InverseJacobian<DataVector, Dim, SourceFrame, TargetFrame>;
+};
+
 /// \ingroup DataBoxTagsGroup
 /// \ingroup ComputationalDomainGroup
 /// Computes the inverse Jacobian of the map held by `MapTag` at the coordinates
 /// held by `SourceCoordsTag`. The coordinates must be in the source frame of
 /// the map.
 template <typename MapTag, typename SourceCoordsTag>
-struct InverseJacobian : db::ComputeTag, db::PrefixTag {
+struct InverseJacobianCompute
+    : InverseJacobian<db::const_item_type<MapTag>::dim,
+                      typename db::const_item_type<MapTag>::source_frame,
+                      typename db::const_item_type<MapTag>::target_frame>,
+      db::ComputeTag,
+      db::PrefixTag {
```

Comment:
why is the compute tag a `db::PrefixTag` but not the simple tag?

Reply:
Presumably because the compute tag is applied to `MapTag`, but I don't think that agrees with the way we originally wanted prefix tags to work. I think we can safely remove `PrefixTag` in this case. What do you think?

Reply:
If it can be removed, I would just remove it.

Reply:
Okay, I added a new commit that removes it